docs/watchlists-feeds-reports.rst

```diff
@@ -0,0 +1,244 @@
+  a large number of network connections.
+* When using ingress IOCs, be careful of errant characters in the ``values`` list, such as leading or trailing
+  whitespace or embedded newline characters.  These errant characters may cause the IOCs to fail, leading to false
+  negative results.
+* ``equality`` IOCs for IPv4 fields (e.g. ``netconn_remote_ipv4``) cannot support CIDR notation; full IP addresses
+  must be used.
+* ``equality`` IOCs for IPv6 fields (e.g. ``netconn_remote_ipv6``) do not support standard or CIDR notation at this
+  time. All IPv6 addresses must omit colon characters, spell out all zeroes in the address, and represent all
+  alphabetic characters in uppercase. For example, "ff02::fb" becomes "FF0200000000000000000000000000FB".
+* Query IOCs must always use field-prefixed queries (key-value pairs); they do not support just searching for a value
+  without a field specified.  Values in query clauses that do not specify fields will be ignored.
+
+:Wrong: ``process_name:chrome.exe AND 192.168.1.1``
+:Right: ``process_name:chrome.exe AND netconn_ipv4:192.168.1.1``
+
+* Query IOCs may search on CIDR address ranges, e.g.: ``netconn_ipv4:192.168.0.0/16``.
+
+Feeds
+-----
+Another way of managing reports is to attach them to a *feed.* Feeds can contain multiple reports, and a feed can be
+attached to a watchlist, effectively making the contents of the watchlist equivalent to the contents of the feed.
+
+A new feed may be created as follows:
+
+::
+
+    >>> from cbc_sdk.enterprise_edr import Feed, Report
+    >>> report = Report(api, None, {"description": "Unsigned processes impersonating browsers",
+    ...                             "iocs_v2": [{
+    ...                                 "id": "unsigned-chrome",
+    ...                                 "match_type": "query",
+    ...                                 "values": [("process_name:chrome.exe NOT process_publisher_state:FILE_SIGNATURE_STATE_SIGNED")]
+    ...                                 }],
+    ...                             "link": None,
+    ...                             "severity": 5,
+    ...                             "tags": ["compliance", "unsigned browsers"],
+    ...                             "timestamp": 1601326338,
+    ...                             "title": "Unsigned Browsers"})
+    >>> feed_data = {'feedinfo': {
+    ...                 'name': 'Suspicious Applications',
+    ...                 'provider_url': 'http://example.com/location',
+    ...                 'summary': 'Any signs of suspicious applications running on our endpoints',
+    ...                 'category': 'external_threat_intel',
+    ...                 'source_label': 'Where the info is coming from'},
+    ...              'reports': [report._info]}
+    >>> feed = api.create(Feed, feed_data)
+
+If you have an existing feed, a new report may be added to it as follows:
```

Comment:
What about replacing or deleting reports?

Reply:
This guide is not intended to address _every single_ possible API that people might use. It's similar to the device control and user management guides in that regard.

Reply:
This are very important workflows for feeds so they should be included and are not complex operations

Reply:
I've included a bit at the end that talks about how to update an existing report, or delete it.